index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare module 'vite-svg-loader' {
   import { Plugin } from 'vite'
-  function svgLoader(options?: { svgoConfig?: Object, svgo?: boolean }): Plugin
+  function svgLoader(options?: { svgoConfig?: import('svgo').OptimizeOptions, svgo?: boolean }): Plugin
```

Comment:
Can you import OptimizeOptions the same way as `Plugin` on line 2?

```js
import { OptimizeOptions } from 'svgo'
```

Reply:
Oops I'm sorry, done.